app/views/layouts/application.html.erb

```diff
@@ -18,13 +18,15 @@
     <%= stylesheet_link_tag    'magnific-popup/magnific-popup.css', 'data-turbolinks-track': 'reload' %>
     <%= stylesheet_link_tag    'ionicons/css/ionicons.min.css', 'data-turbolinks-track': 'reload' %>
     <%= stylesheet_link_tag    'fullpage/fullpage.css', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag 'quill/quill.snow.css', 'data-turbolinks-track': 'reload' %>
```

Comment:
Note: It is quill.snow.css here because we used the 'snow' theme. We can use other themes as well but I only included snow here.

Reply:
Alternatively, we can remove the other themes from stylesheets/quill and only have a single css there,